### PR TITLE
Diagrama de causa e efeito

### PR DESCRIPTION
**Pull request by Thais-ra**
Artefato sobre o diagrama de ishikawa contendo o diagrama feito pelo grupo e a elaboração do mesmo usando a ferramenta draw.io

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the CONTRIBUTING.md doc.
- [x] I have added necessary documentation (if appropriate).
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: docs/Base/causaEfeito.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Diagrama de Causa e Efeito
+
+## Histórico de versões
+| Data     | Versão | Descrição            | Autor(a)                                      | Revisor(a) |
+| -------- | ------ | -------------------- | --------------------------------------------- | ---------- |
+| 20.06.22 | 1.0    | Criação do documento | [Thaís Rebouças](https://github.com/Thais-ra) | -          |
```
Comment (AmandaNbr) [inline docs/Base/causaEfeito.md:6]: Lemrar de colocar o revisor

File: docs/Base/causaEfeito.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- [Yudi Yamane](https://github.com/yudi-azvd)
+
+## Resultados
+
+Durante o brainstorming a equipe utilizou uma categoria que foi retirada na versão final do diagrama, a categoria "Meio Ambiente" que foi substituída por "Medida".
+
+![Figura01](https://user-images.githubusercontent.com/35047444/174613534-c71485bf-b963-4456-b72c-95d82703e952.jpeg)
+<p align = "center"> 
+Figura 01 - Ishikawa feito por meio de brainstorming <br>
+</p>
```
Comment (AmandaNbr) [inline docs/Base/causaEfeito.md:47]: Colocar os autores

File: docs/Base/causaEfeito.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+![Figura02](https://user-images.githubusercontent.com/35047444/174613340-2c774cfa-f207-4de9-aa7c-59fe2fcac055.png)
+<p align = "center"> 
+Figura 02 - Versão 01 do diagrama de Ishikawa <br>
+Autora: Thaís 
+</p>
+
+## Referências
+
```
Comment (AmandaNbr) [inline docs/Base/causaEfeito.md:56]: Colocar referência

**Review by AmandaNbr — CHANGES_REQUESTED** [review]

File: docs/Base/causaEfeito.md (hunk 2 above)
Comment (Thais-ra) [reply docs/Base/causaEfeito.md:47]: Precisa? já que tem todos os participantes logo em cima

**Review by AmandaNbr — APPROVED** [review]

**Pull request merged**